TranslationUploader/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Suggestion - can't we choose this file by uploader control ?
Then we would be able to use it for every our project without any changes.